src/odemis/acq/stream/_projection.py

```diff
@@ -1660,7 +1660,11 @@ def _updateImage(self):
         try:
             data = self._computeSpec()
             if data is not None:
-                self.image.value = self._project2RGB(data, self.stream.tint.value)
+                tint = self.stream.tint.value
+                if tint == TINT_FIT_TO_RGB:
+                    # Not supported for this view => fallback to greyscale
+                    tint = (255, 255, 255)
+                self.image.value = self._project2RGB(data, tint)
```

Comment:
self._project2RGB is used in other parts of this file, do we need this change only here and not in the rest?

Reply:
Good catch. "Fit-To-RGB" can be set on any spectrum stream. So we need to also protect `PixelTemporalSpectrumProjection` .